addon/components/tf-form/component.js

```diff
@@ -0,0 +1,30 @@
+/**
+  @module ember-ticketfly-fields
+ */
+import Ember from 'ember';
+import layout from './template';
+const { Component, guidFor, computed } = Ember;
+
+/**
+  @public
+  @class TfForm
+  @extends Ember.Component
+ */
+export default Component.extend({
+  layout,
+  
+  tagName: 'form',
+  classNames: ['tf-form'],
+
+  inputGuid: computed({
+    get() {
+      return `id-${guidFor(this)}`;
+    }
+  }),
+
+  describedByGuid: computed({
+    get() {
+      return `described-by-${guidFor(this)}`;
+    }
+  })
+});
```

Comment:
@BrianSipple Sounds good. I think changing `tf-form` to what you posted makes sense. I'd go with `tf-form-group` instead of *field*, and a `div` instead of `p`.